website/blog/2024-02-29-cortex-slack.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "LLM-powered Analytics Engineering: How we're using AI inside of our dbt project, today, with no new tools."
+description: "With Snowflake's new Cortex functions and the dbt Cloud orchestrator, we can quickly and safely do analytics that was once impractical."
+slug: dbt-models-with-snowflake-cortex
+
+authors: [joel_labes]
+
+tags: [analytics craft, data ecosystem]
+hide_table_of_contents: false
+
+date: 2024-03-04
+is_featured: true
+---
+
+## Cloud Data Platforms make new things possible; dbt helps you put them into production
```

Comment:
This is a very long header and to me - disrupts the flow quickly. Could we slip into it with more premise?

Reply:
I normally skim blogs and so I want the headers to be big takeaways that I want people to get even if they don't read the rest of the section. So I'd rather keep this as-is